ci/ansible/roles/pulp-crane/files/pulp_crane_el6.conf

```diff
@@ -0,0 +1,32 @@
+# Use this config with Apache 2.2 or earlier
+<VirtualHost *>
+    WSGIScriptAlias / /usr/share/crane/crane_el6.wsgi
+    <Location /crane>
+        Order Deny,Allow
+        Deny from all
+        Allow from localhost
+    </Location>
+    <Directory /usr/share/crane/>
+        Order allow,deny
+        Allow from all
+    </Directory>
+</VirtualHost>
+
+# <VirtualHost *:443>
```

Comment:
I'm fine with that. i just grabbed this file from pulp crane:
https://github.com/pulp/crane/blob/master/deployment/apache22.conf